websites/M/Moopa/metadata.json

```
@@ -0,0 +1,38 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "338295424479789057",
+		"name": "factiven"
+	},
+	"service": "Moopa",
+	"description": {
+		"en": "Moopa is a free Anime and Manga streaming website with AniList integration where you can easily track your progress with a single click of a button."
+	},
+	"url": "moopa.live",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/lelaLhp.png",
+	"thumbnail": "https://i.imgur.com/pNmZsH3.png",
+	"color": "#141519",
```

This color is the base color used when something is selected or hovered on
```suggestion
	"color": "#FF7F57",
```